scripts/spack/packages/raja/package.py

```diff
@@ -111,14 +111,22 @@ def initconfig_hardware_entries(self):
             entries.append(cmake_cache_option("ENABLE_CUDA", False))
 
         if '+rocm' in spec:
```

Comment:
Should these changes be upstreamed to raja and spack? or is this PR downstreaming them from those repos?

Tag: @white238 @rhornung67

Reply:
This PR is downstreaming changes from [RAJA's spack repository](https://github.com/LLNL/RAJA/blob/8e8cc96cbccda2bfc33b14b57a8591b2cf9ca342/scripts/spack_packages/raja/package.py#L296).